src/assets/specs/README.md

```diff
@@ -0,0 +1,42 @@
+## Generators
+
+### To generate go code:
+
+oapi-codegen src/assets/specs/command-api.yaml > api/commands.go
+
+#### Dependencies:
+
+* opai-codegen: To install:
+
+```bash
+go get github.com/deepmap/oapi-codegen/cmd/oapi-codegen
+```
+
+### To generate documentation:
+
+I looked at various tools. They either wanted me to sign up or didn't work:
```

Comment:
This doesn't seem to belong to a checked-in file (maybe a commit message).


(I'm just exploring by dumping the GitHub raw view URL into https://petstore.swagger.io/)

Reply:
Good point. I would have liked a non-docker solution with a single install to just work without friction, but didn't find anything else like that.